stable/mlrun-kit/templates/_helpers.tpl

```diff
@@ -20,12 +20,16 @@ Create fully qualified names.
 {{- end -}}
 {{- end -}}
 
-{{- define "mlrun-kit.jupyter.fullname" -}}
-{{- printf "%s-jupyter"  (include "mlrun-kit.fullname" .) | trunc 63 | trimSuffix "-" -}}
+{{- define "mlrun-kit.shared-storage-pvc.fullname" -}}
```

Comment:
Please rename shared-storage to `shared-persistency` everywhere (so it would be consistent with `mlrun-persistency`)